rpcservers/wallet/walletkit_server.go

```diff
@@ -64,90 +65,132 @@ var (
 
 	// macPermissions maps RPC calls to the permissions they require.
 	macPermissions = map[string][]bakery.Op{
-		"/walletrpc.WalletKit/DeriveNextKey": {{
-			Entity: "address",
-			Action: "read",
-		}},
-		"/walletrpc.WalletKit/DeriveKey": {{
-			Entity: "address",
-			Action: "read",
-		}},
-		"/walletrpc.WalletKit/NextAddr": {{
-			Entity: "address",
-			Action: "read",
-		}},
-		"/walletrpc.WalletKit/PublishTransaction": {{
-			Entity: "onchain",
-			Action: "write",
-		}},
-		"/walletrpc.WalletKit/SendOutputs": {{
-			Entity: "onchain",
-			Action: "write",
-		}},
-		"/walletrpc.WalletKit/EstimateFee": {{
-			Entity: "onchain",
-			Action: "read",
-		}},
-		"/walletrpc.WalletKit/PendingSweeps": {{
-			Entity: "onchain",
-			Action: "read",
-		}},
-		"/walletrpc.WalletKit/BumpFee": {{
-			Entity: "onchain",
-			Action: "write",
-		}},
-		"/walletrpc.WalletKit/ListSweeps": {{
-			Entity: "onchain",
-			Action: "read",
-		}},
-		"/walletrpc.WalletKit/LabelTransaction": {{
-			Entity: "onchain",
-			Action: "write",
-		}},
-		"/walletrpc.WalletKit/LeaseOutput": {{
-			Entity: "onchain",
-			Action: "write",
-		}},
-		"/walletrpc.WalletKit/ReleaseOutput": {{
-			Entity: "onchain",
-			Action: "write",
-		}},
-		"/walletrpc.WalletKit/ListLeases": {{
-			Entity: "onchain",
-			Action: "read",
-		}},
-		"/walletrpc.WalletKit/ListUnspent": {{
-			Entity: "onchain",
-			Action: "read",
-		}},
-		"/walletrpc.WalletKit/FundPsbt": {{
-			Entity: "onchain",
-			Action: "write",
-		}},
-		"/walletrpc.WalletKit/SignPsbt": {{
-			Entity: "onchain",
-			Action: "write",
-		}},
-		"/walletrpc.WalletKit/FinalizePsbt": {{
-			Entity: "onchain",
-			Action: "write",
-		}},
-		"/walletrpc.WalletKit/ListAccounts": {{
-			Entity: "onchain",
-			Action: "read",
-		}},
-		"/walletrpc.WalletKit/RequiredReserve": {{
-			Entity: "onchain",
-			Action: "read",
-		}},
-		"/walletrpc.WalletKit/ImportAccount": {{
-			Entity: "onchain",
-			Action: "write",
-		}},
-		"/walletrpc.WalletKit/ImportPublicKey": {{
-			Entity: "onchain",
-			Action: "write",
-		}},
+		"/walletrpc.Kit/DeriveNextKey": {
```

Comment:
This is the reason for your "unknown permission" failure. You can't rename these.

Reply:
Oh, great, I most likely did not intentionally do that. Goland does renames by default including strings. I will examine all of these strings and make sure they are same as previous. Thankyou.

Reply:
The names of the actual variables were changed to fit the new naming scheme. After going through all of them it looks like that was the only ones renamed, makes sense since they were the ones with the worst names as variables. walletkit.WalletKit. Not sure... should I change that back completely?

Reply:
Yeah, please don't rename the structs as they are lined up with what the RPC services are called. Also, please try to extract some of the changes (e.g. exporting some of the functions/methods) into their own commits. And remove all non-necessary formatting changes (see comment below). Nobody will seriously review a commit that has 3200 lines of change.

Reply:
I only changed them on the prompting of the linter.